Package.swift

```diff
@@ -29,7 +29,7 @@ let package = Package(
     ],
     dependencies: [
         // Dependencies declare other packages that this package depends on.
-        .package(url: "https://github.com/IBM-Swift/Kitura-NIO.git", from: "0.0.0"),
+        .package(url: "https://github.com/IBM-Swift/Kitura-NIO.git", .branch("master")),
```

Comment:
@pushkarnk  This is a temporary change since we don't have any latest release for Kitura-NIO